build_raven

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# fail if anything here fails
+set -e
+
+RAVEN_BUILD_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+echo Making RAVEN ...
+
+# defaults
+MAKE_ARGS=" "
+LIBS_MODE=0 # 0 is "create", 1 is "load"
+#### FUTURE WORK:
+####   combine "make" and "establish_conda_env.sh" into one installation command
+ECE_ARGS=" "
+
+# read arguments
+while test $# -gt 0
+do
+  case "$1" in
+     --coverage)
+       MAKE_ARGS="${MAKE_ARGS} coverage=true graceful=true"
+       LIBS_MODE=1
+       ;;
+     --jobs) MAKE_ARGS="-j $1 ${MAKE_ARGS}"
+#    --help)     ECE_ARGS="${ECE_ARGS} --help"; ;;
+#    --optional) ECE_ARGS="${ECE_ARGS} --optional"; ;;
+#    --no-clean) ECE_ARGS="${ECE_ARGS} --no-clean"; ;;
+#    --conda-defs)
+#      shift
+#      ECE_ARGS="${ECE_ARGS} --conda-defs $1"
+#      ;;
+#    --raven-libs-name)
+#      shift
+#      export RAVEN_LIBS_NAME="$1"
+#      ;;
```

Comment:
The comments lines are for the future work?

Reply:
Yes, if it turns out we don't need to environment variable, it would be nice to work all through the command line in the future; however, this option didn't work for an undetermined reason previously.